pkg/resource/logging-config/logging-config.go

```diff
@@ -61,3 +65,21 @@ func ConfigMeta(lc loggedcluster.Interface) metav1.ObjectMeta {
 func getLoggingConfigName(lc loggedcluster.Interface) string {
 	return fmt.Sprintf("%s-%s", lc.GetClusterName(), loggingConfigName)
 }
+
+func listTenants(k8sClient client.Client, ctx context.Context) ([]string, error) {
+	tenantList := make([]string, 0)
+	var grafaOrgList v1alpha1.GrafanaOrganizationList
+
+	err := k8sClient.List(ctx, &grafaOrgList)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, grafanaOrg := range grafaOrgList.Items {
+		for _, tenant := range grafanaOrg.Spec.Tenants {
+			tenantList = append(tenantList, string(tenant))
```

Comment:
Can we make sure this list is unique? Maybe we do not care 🤔

Reply:
:heavy_check_mark: